doc/source/tune-usage.rst

```diff
@@ -457,29 +457,43 @@ be wrapped with ``tune.function``):
     )
 
 
-Client API
-----------
-
-You can modify an ongoing experiment by adding or deleting trials using the Tune Client API. To do this, verify that you have the ``requests`` library installed:
-
-.. code-block:: bash
+Tune REST API
+-------------
 
-    $ pip install requests
+You can interact with an ongoing experiment with the Tune API. The Tune API is organized around REST, which includes resource-oriented URLs, accepts form-encoded requests, returns JSON-encoded responses, and uses standard HTTP protocol.
 
-To use the Client API, you can start your experiment with ``with_server=True``:
+To allow Tune to receive and respond to your API calls, you have to start your experiment with ``with_server=True``:
 
 .. code-block:: python
 
     run_experiments({...}, with_server=True, server_port=4321)
 
-Then, on the client side, you can use the following class. The server address defaults to ``localhost:4321``. If on a cluster, you may want to forward this port (e.g. ``ssh -L <local_port>:localhost:<remote_port> <address>``) so that you can use the Client on your local machine.
+The easiest way to use the Tune API is with the built-in TuneClient. To use TuneClient, verify that you have the ``requests`` library installed:
```

Comment:
yeah - to be clear, the Tune API is everything that is in `ray.tune` - we should be clear about this section being a Tune Client API (Tune REST API is also fine)

Reply:
@adizim